File: M. Rivqi Al Varras_22004_source code unit IV/japerhari.py
print ("jumlah hari pada suatu bulan pada tahun tertentu")
print (".........................................")
hari = [31,28,31,30,31,30,31,31,30,31,30,31]
bulan=int(input("Masukan Bulan ke- (1-12) : "))
tahun=int(input("Masukan Tahun: "))
if(bulan == 2):
    if((tahun % 4 == 0) & (tahun % 100 !=0)):
     jumlah_hari = 29
    else:
        jumlah_hari = 28
else:jumlah_hari = hari [bulan-1]
print("\nJumlah hari = ", jumlah_hari)

File: M. Rivqi Al Varras_22004_source code unit IV/japerhari.c
#include <stdio.h>
int main()
{
int bulan, tahun, jumlahhari;
//inisialisasi array
int jum_hari[12] = {31, 28, 31, 30, 31, 30, 31, 31,30, 31, 30, 31};
printf("----------jumlah hari pada suatu bulan pada tahun tertentu----------\n");
printf("-------------Inputkan Salah satu Bulan dan Tahun------------------\n\n");
printf("==================================================================\n\n");
printf("Masukkan bulan (1..12)\t: ");
scanf("%d", &bulan);
printf("Masukkan tahunnya\t: ");
scanf("%d", &tahun);
//cek tahun kabisat
if(bulan == 2)
{
if(tahun % 4 == 0) {
jumlahhari = 29;
} else {
jumlahhari = 28;
}
} else {
jumlahhari = jum_hari[bulan-1];
}
printf("\nJumlah hari dalam bulan %d tahun %d adalah %d hari\n",bulan,tahun,jumlahhari);
}